.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint, Test & Build
+on:
+  pull_request:
+    branches:
+      - "development"
```

Comment:
I think your dev branch is 'develop' not 'development'. So this configuration won't work with the current setup.